smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolServiceClient.kt

```diff
@@ -122,6 +122,9 @@ open class HttpProtocolServiceClient(
                 .flatMap { it.getProperties(ctx) }
                 .let { overrideConfigProperties(it) }
                 .sortedBy { it.accessModifier }
+                .mapNotNull {
+                    customizedClientConfigProperty(it)
+                }
```

Comment:
Adds a customization point to allow for modification of client config fields, i.e. those copied in from endpoint params